### PR TITLE
Data model

### PR DESCRIPTION
**Pull request by chimericalidea**
a stab at converting the #11 into something helpful. Let me know if there are other "key insights" that we should highlight for the user. It doesn't seem to be helpful to explain to the user how we created the JSON, so I didn't explain that. If you disagree, we obviously can. :)

*** this includes the changes in #14 . do not merge unless you want to make everything live.

### CODE REVIEW

**Comment by abesto** [comment]
@adriancole how do you feel about this in its current state?
@chimericalidea since this PR was opened some extra information (even graphs!) was added to #11. Do you think it makes sense to incorporate some of those?

File: pages/architecture.md [hunk 1]
```diff
@@ -4,53 +4,50 @@ weight: 2
...
+
+![Architecture overview]({{ site.github.url }}/public/img/architecture-0.png)
 
 Transport
 ---------
 
-Spans must be transported from the services being traced to Zipkin collectors.
-There are two primary transports, Scribe and Kafka. Scribe is deprecated.
+Spans sent by the instrumented library must be transported from the services being traced to Zipkin collectors.
+There are two primary transports: Scribe and Kafka. Scribe is shown in the figure. See [Span Receivers]({{ site.github.url }}/pages/span_receivers) for more information.
```
Comment (adriancole) [inline pages/architecture.md]: HTTP is also a primary transport

File: pages/architecture.md [hunk 2]
```diff
@@ -4,53 +4,50 @@ weight: 2
...
 
-Storage
--------
+### Storage
 
-We originally built Zipkin on Cassandra for storage. It's scalable, has a
+Zipkin was initially built to store data on Cassandra since Cassandra is scalable, has a
 flexible schema, and is heavily used within Twitter. However, we made this
-component pluggable, and we now have support for Redis and MySQL.
+component pluggable. In addition to Cassandra, we support Redis and MySQL.
```
Comment (adriancole) [inline pages/architecture.md]: Redis has been abandoned. Elasticsearch support exists.

File: pages/data_model.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
+---
+title: Data Model
+---
+
+In order to illustrate the tracing data that Zipkin displays, let's relate it to the equivalent information in the Zipkin data model. By comparing these, we see that
```
Comment (adriancole) [inline pages/data_model.md]: This is a good start

**Comment by adriancole** [comment]
LGTM thanks!

**Pull request merged**